src/ui/view/dataverse/component/tags/tags.tsx

```diff
@@ -0,0 +1,15 @@
+import type { FC } from 'react'
+import { Tag } from '@/ui/component/tag/tag'
+import './tags.scss'
+
+type TagsProps = {
+  tags: string[]
+}
+
+export const Tags: FC<TagsProps> = ({ tags }): JSX.Element => (
+  <div className="okp4-dataverse-portal-dataverse-component-description-tags">
+    {tags.map((tag: string) => (
+      <Tag key={`dataset-tags-${tag}`} tagName={tag}></Tag>
```

Comment:
I'm not sure why `dataset` is specified here

Reply:
Also, isn't `Tag` self closing?

Reply:
Both are fixed in [d39c77d](https://github.com/okp4/dataverse-portal/pull/146/commits/d39c77d87463afd79a0b3a61bead8cc748c3c1fb).